test/b_capped_sto.js

```diff
@@ -837,6 +837,19 @@ contract("CappedSTO", async (accounts) => {
             });
         });
 
+        describe("Pricing Test cases for Module Factory", async () => {
+            it("Should return correct price when price is in poly", async () => {
+                let newFactory = await CappedSTOFactory.new(
```

Comment:
I don't think this test full filling its `it statement`.

Reply:
Not sure where the asserts went lol. I'll add them back.